device-types/Datacom/DM4618-32GPON-plus-2CX-plus-4VS.yaml

```diff
@@ -0,0 +1,115 @@
+---
+manufacturer: Datacom
+model: DM4618 32GPON+2CX+4VS
+slug: datacom-dm4618-32gpon-plus-2cx-plus-4vs
+part_number: 800.5198.xx
+u_height: 2.0
+is_full_depth: true
+airflow: side-to-rear
+comments: "OLT with 32 GPON ports, 2 ports at 40/100Gbit/s in\nQSFP28, 4 optical ports at 25/10 Gigabit Ethernet in SFP+,\n1 Slot for Interface Module.\
+  \ Removable ventilation modules included in the equipment. Two slots for redundant and hot-swappable AC or DC power supplies.\nPower supplies and SFPs\
+  \ must be purchased separately."
+weight: 8.25
+weight_unit: kg
+console-ports:
+  - name: Console0
+    type: rj-45
+  - name: Console1
+    type: usb-a
+console-server-ports:
+  - name: Alarms
+    type: rj-45
+    description: 2 inputs and 1 output
```

Comment:
What is this actually? Is this actually used as a console server port? If not, please remove this definition as it would be out of scope.

Reply:
removed